src/test/java/com/rocketden/main/service/RoomServiceTests.java

```diff
@@ -213,36 +215,37 @@ public void changeRoomHostSuccess() {
 
     @Test
     public void changeRoomHostFailure() {
-        String roomId = "012345";
         Room room = new Room();
-        room.setRoomId(roomId);
+        room.setRoomId(ROOM_ID);
 
         User host = new User();
-        host.setNickname("host");
+        host.setNickname(NICKNAME);
         User user =  new User();
-        user.setNickname("user");
+        user.setNickname(NICKNAME_2);
 
         room.setHost(host);
         room.addUser(host);
         room.addUser(user);
 
-        Mockito.doReturn(room).when(repository).findRoomByRoomId(eq(roomId));
+        Mockito.doReturn(room).when(repository).findRoomByRoomId(eq(ROOM_ID));
 
         // Invalid permissions
         UpdateHostRequest invalidPermRequest = new UpdateHostRequest();
         invalidPermRequest.setInitiator(UserMapper.toDto(user));
         invalidPermRequest.setNewHost(UserMapper.toDto(host));
 
-        ApiException exception = assertThrows(ApiException.class, () ->
-                roomService.updateRoomHost(room.getRoomId(), invalidPermRequest));
-        assertEquals(RoomError.INVALID_PERMISSIONS, exception.getError());
+        try {
+            roomService.updateRoomHost(room.getRoomId(), invalidPermRequest);
+        } catch (ApiException e) {
+            assertEquals(RoomError.INVALID_PERMISSIONS, e.getError());
+        }
```

Comment:
What happens if the `updateRoomHost` call succeeds without throwing an error? 

Also out of curiosity, what was the warning you got with the `assertThrows` here (I remember I also had some warning here that I fixed when writing these, so maybe this one is similar)? Main reason I ask is cause if there's an easy fix then I might prefer keeping the `assertThrows` over using try/catch blocks for cleanliness and that (potential) bug mentioned above

Reply:
Attached is an image of the `assertThrows(...)` error; I think I actually misinterpreted it, so thanks for bringing this up. The error is that there were multiple method calls, and it wasn't clear which one threw the error -- I've updated these lines to avoid the errors while not doing `try / catch`.

<img width="528" alt="AssertThrows Error" src="https://user-images.githubusercontent.com/7987279/96375802-0d0d2600-1130-11eb-93a6-102c8d4befff.png">

Reply:
Fixed! Just had to replace `room.getRoomId()` with `ROOM_ID` in all instances.